<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrganizationGroupItemWidget</class>
 <widget class="QWidget" name="OrganizationGroupItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>47</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OrganizationGroupItemWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1,0">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="labelGroup">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string>group</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelLoading">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string>loading</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelName">
     <property name="text">
      <string>name</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
